### DISCUSSION TITLE
Newbie Needs Info On Companion App

### DISCUSSION THREAD

**Discussion by Wanderinpilgrim** [aclap-dev/video-downloadhelper#907] [Help and Q&A]
### Your question

HI, I have your extension installed in chrome and now in brave. I often get the propmt to install the companion app with ertain streams i attempt to download.
Yours is the only chrome extension that is not 'all inclusive'. I guess i'm scared to download it bc i don't know what i'm doing.
If the Coapp cannot be included in the extension and if i will have to download it as a separate thing, please talk to me. 
Is this where a fee come in? Where on my windows 10 PC will the app reside? Once installed, will it work seamlessly in the background - or can i just navigate the extension as i'm used to doing, only now those undownloadable streams will now miraculously download? IOW, i don't know what i'm getting into and i fear it will be a complicating procedure to use(not tech saavy). If you could please shed some light, that'd be great.
Thanks!

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: en-US
coapp: {"error":"Specified native messaging host not found.","found":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 1
}

**Comment by GrampaWildWilly** [comment]
#744
